pytermgui/markup/parsing.py

```diff
@@ -189,120 +188,191 @@ def tokenize_markup(text: str) -> Iterator[Token]:
         yield PlainToken(text[cursor:length])
 
 
-def tokenize_ansi(  # pylint: disable=too-many-locals, too-many-branches, too-many-statements
-    text: str,
+def _tokenize_ansi_color(
+    params: List[str],
```

Comment:
You can use `list[str]` here as we have `__annotations__` imported for lower Python versions.